examples/space.html

```diff
@@ -43,19 +43,33 @@
     fsButton.addEventListener('click', function() {
       scene.enterVR();
     });
+
     var cubes = document.querySelectorAll('vr-cube');
     var i;
     for (i = 0; i < cubes.length; ++i) {
       cubes[i].addEventListener('click', function(evt) {
```

Comment:
This click here interferes with the mouse look.    Will use pointerlock to resolve this issue in a later PR.